scripts/report_scripts/qc_report_per_sample.Rmd

```diff
@@ -61,10 +59,9 @@ Coverage <- data.frame("Total coverage" = coverage.df[4],
 datatable(t(Coverage), options = list(searching = FALSE, paging = FALSE))
 
 ```
-## FastQC and Trimming
+## MultiQC and Trimming
 
-Download: [FastQC report of unprocessed data](`r params$fastqc_unprocessed`)
-Download: [FastQC report of pre-processes data](`r params$fastqc_processed`)
+Download: [MultiQC report of raw and processed data](`r file.path('multiqc', params$sample_name, 'multiqc_report.html')`)
```

Comment:
I'd prefer if this was provided as an input and thus would be accessed via `params`.  This ensures that snakemake is aware of the reference to this file, which is the product of another rule.

Reply:
Can we - aka I - apply that change so we can merge that tonight?

Reply:
the multiqc file is given as input for that qc_report rule. It's just not used as an argument because the multiqc reports are in some sample specific subdirs of output/reports and not just there which. So e.g. "basename(params$multiqc)" can't bes used as it is used to link to the krona reports